database/factories/ModelFactory.php

```diff
@@ -3,7 +3,7 @@
 namespace VendorName\Skeleton\Database\Factories;
 
 /*
-class ModelFactory extends Factory
+class ModelFactory extends \Illuminate\Database\Eloquent\Factories\Factory
```

Comment:
Can we import this with `use` instead or is that not possible?

Reply:
It was originally, but it gets stripped because the factory is commented out and we have a rule to remove unused `use` statements.

Reply:
Yes... 🤦🏻 D'oh.